scripts/lib/mkcloud-driver-libvirt.sh

```diff
@@ -122,6 +131,7 @@ function libvirt_do_setuphost()
 
     sed -i 's/net.ipv4.ip_forward = 0/net.ipv4.ip_forward = 1/' /etc/sysctl.conf
     echo "net.ipv4.conf.all.rp_filter = 0" > /etc/sysctl.d/90-cloudrpfilter.conf
+    echo "net.ipv6.conf.all.forwarding = 1" > /etc/sysctl.d/90-cloudipv6.conf
```

Comment:
No, bad! I missed this first time around. And in fact, libvirt needs forwarding set to 2 for the primary interface.

Reply:
oh yes, and it's on setup host too, so _definitely_ something that can be done as a host setup.

Reply:
This is just setting up forwarding not accept_ra, I thought it was only boolean (0 or 1). 
However I agree setting in only on the main interface makes sense. though they are setting it globally for ipv4